docs/docs/app/data-source-types/missing-datasource-docs.mdx

```diff
@@ -0,0 +1,14 @@
+import Link from '@docusaurus/Link';
+
+export default function MissingDatasourceDocs({ datasource, slug }) {
+    return (
+        <>
+            <p>
+                Unfortunately, even though Growthbook fully works with {datasource}, a How-To guide for {datasource} hasn't been written yet.  We do have a thriving developer community on <Link to={`https://slack.growthbook.io?ref=docs-datasource-${slug}`}>Slack</Link> that you can reach out to in order to get your implementation working. 
```

Comment:
nit: my lazy brain read this and it seemed like we didn't support this when I first scanned it. Maybe start with a separate paragraph:

"Growthbook works with {datasource}!"

Then after a line break

"Unfortunately, a How-To..."

Reply:
I've fixed this and changed the templates a bit more.